### DISCUSSION TITLE
@rspress/plugin-playground 的 monaco 编辑器如何支持 TSX 语法识别

### DISCUSSION THREAD

**Discussion by chhpt** [web-infra-dev/rspress#171] [Q&A]
如图，使用 @rspress/plugin-playground，会出现语法错误的提示
<img width="876" alt="image" src="https://github.com/web-infra-dev/rspress/assets/19288423/9d4ed372-266e-419f-a109-a4f19ddc7878">
<img width="788" alt="image" src="https://github.com/web-infra-dev/rspress/assets/19288423/ae5eabbd-4960-4798-a631-a823900309aa">

**Comment by sanyuan0704** [comment]
playground 的编辑器暂不支持类型：
![image](https://github.com/web-infra-dev/rspress/assets/39261479/b5deac74-7bc2-40e5-86a9-cc199eb2537e)
